test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,61 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ClientServiceFixtures {
+
+  private static Client client1 = new Client();
+  private static Client client2 = new Client();
+  private static Client client3 = new Client();
+  private static Client client4 = new Client();
+  public static final Long FAKEID = Long.valueOf(10);
+
+  public static Client getClient1() {
+    buildClient(client1, Roles.CLIENT, 21, "Diana", "dxmd", "Monegro", StatusEnum.ACTIVE);
```

Comment:
You don't need any `getClient#` because you already have a `buildClient`

Reply:
solved